mindmap/mindmap.py

```diff
@@ -44,6 +44,13 @@ class MindMapXBlock(XBlock):
         scope=Scope.settings,
     )
 
+    is_static = Boolean(
+        help="Whether the mind map is static or not",
+        display_name="Is Static",
```

Comment:
I'm can't decide between `Is static?` or `Is a static mindmap?`

Reply:
Maybe, `Is a static mind map` is a little clearer for the user?